OSDOCS-12318: adds bootc image use to MicroShift (#90445)
openshift/openshift-docs: modules/microshift-install-bootc-workflow.adoc

// Module included in the following assemblies:
//
// microshift_install_bootc/microshift-about-rhel-image-mode

:_mod-docs-content-type: CONCEPT
[id="microshift-install-rhel-image-mode-conc_{context}"]
= The image mode for {op-system-base} with {microshift-short} workflow

To use image mode for {op-system-base}, ensure that the following resources are available:

> * A {op-system-base} {op-system-version} host with an active Red{nbap}Hat subscription for building {microshift-short} bootc images.

Reviewer: There are two problems with this sentence:
- `{nbap}` is not rendered correctly
- The RHEL version is rendered to 9.6 - it should be 9.4 in 4.18.

* A remote registry for storing and accessing bootc images.
* You can use image mode for RHEL with a {microshift-short} cluster on aarch64 or x86_64 system architectures.

The workflow for using image mode with {microshift-short} includes the following steps:

. Find and use a prebuilt {microshift-short} container image to install {op-system-base-full}.
. Build a custom {microshift-short} container image if the prebuilt {microshift-short} container image requires customization.
. Run the container image.

[IMPORTANT]
====

> The `rpm-ostree` file system is not supported in image mode and must not be used to make changes to deployments that use image mode.

Reviewer 2: ```suggestion
The `rpm-ostree` file system is not supported in image mode. Do not use the `rpm-ostree` file system to modify deployments that use image mode.
```

This is up to you. Apologies if I nitpicked.

Author: That's a good call. It will translate better.
